### PR TITLE
[New] Install drupal 1k

### PR DESCRIPTION
**Pull request by leslitagordita**
* Original PR: https://github.com/linode/docs/pull/3143 (author: @Rajakavitha1) . Submitted new PR to fix some Git commit history issues.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit c273edca48183a420bc791280e1e47ca69e60d5d

https://deploy-preview-3151--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-centos-8/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo yum install -y php php-{cli,mysqlnd,json,opcache,xml,mbstring,gd,curl}
+
+1. Create your Drupal 8 installation's `settings.php` file from the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
+
+        sudo cp /var/www/html/example.com/public_html/sites/default/default.settings.php /var/www/html/example.com/public_html/sites/default/settings.php
+
+1.  Enforce [trusted hostnames](https://www.drupal.org/node/2410395) with those that users will access your site from. With the text editor of your choice, edit your `settings.php` file replacing the values with your own site's URL(s).
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-centos-8/index.md]: updated

File: docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-debian-10/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo apt-get install php-gd php-xml php-dom php-Simplexml php-mbstring
+
+1. Create your Drupal 8 installation's `settings.php` file from the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
+
+        sudo cp /var/www/html/example.com/public_html/sites/default/default.settings.php /var/www/html/example.com/public_html/sites/default/settings.php
+
+1.  Enforce [trusted hostnames](https://www.drupal.org/node/2410395) with those that users will access your site from. With the text editor of your choice, edit your `settings.php` file replacing the values with your own site's URL(s).
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-debian-10/index.md]: updated

File: docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-ubuntu-18-04/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo apt-get install php-gd php-xml php-dom php-Simplexml php-mbstring
+
+1. Create your Drupal 8 installation's `settings.php` file from the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
+
+        sudo cp /var/www/html/example.com/public_html/sites/default/default.settings.php /var/www/html/example.com/public_html/sites/default/settings.php
+
+1.  Enforce [trusted hostnames](https://www.drupal.org/node/2410395) with those that users will access your site from. With the text editor of your choice, edit your `settings.php` file replacing the values with your own site's URL(s).
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-ubuntu-18-04/index.md]: updated

**Pull request merged**